src/how-to-stacker.md

```diff
@@ -1 +1,72 @@
 # How to Participate as a Stacker
+Participating as a Stacker in the Stacks Blockchain is an essential role that involves validating and adding new sBTC transactions to the blockchain. As a Stacker, you are responsible for ensuring the accuracy and integrity of these transactions. To become a Stacker, you must hold and temporarily lock STX, Stacks’ native currency, and support the network’s security and consensus. As a reward, you earn BTC. 
```

Comment:
Nit: I think the first sentence can cause some confusion since we don't specify which sBTC transactions and which of the blockchains they are added to. I'd suggest just claiming that stackers are essential for the liveliness of sBTC. We can leave the details to the "Stacker responsibilities" page.

```suggestion
Participating as a Stacker in the Stacks Blockchain is an essential role to ensure the liveliness of sBTC. To become a Stacker, you must hold and temporarily lock STX, Stacks’ native currency, and support the network’s security and consensus. As a reward, you earn BTC. 
```